src/modulePage/ModuleSection.tsx

```diff
@@ -67,7 +76,7 @@ function getDemo(comp, scheme) {
         case "HaarWaveletDemo": 
             return <ImageSelectableDemo Demo={HaarWaveletDemo} initImg='bwWoman.jpg' demoProps={demoArgs}/>
         case "PCADemo":
-            return <PCADemo {...demoArgs}/>
+            return <PCADemo {...demoArgs} labelColorHex=''/>
```

Comment:
If 'labelColorHex' isn't used, I'd define a default prop in the `PCADemo` component or omit it from that component entirely.

Reply:
Nice, didn't know you could do this